src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,53 @@
+.icon {
+  display: block;
+  width: $icon-width;
+  padding-bottom: $icon-width;
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  text-decoration: none;
+
+  &--menu {
+    background-image: url(../images/menu.svg);
+  }
+
+  &--menu-close {
+    background-image: url(../images/menu-close.svg);
+  }
+
+  &--call {
+    background-image: url(../images/phone.svg);
+
+    margin-right: 30px;
+    text-decoration: none;
+
+    @include onTablet {
+      margin-right: 40px;
+      width: 24px;
+      height: 24px;
+    }
+  }

```

Comment:
on mobile 
 to match layout you need to change icons width to 18px

Reply:
don't see where are icons with width 18px on the layout
![image](https://user-images.githubusercontent.com/116519764/210060506-00c1160e-9238-49e7-a0a3-a5c7970af8a0.png)

Reply:
agree, size on your page changes correct

Reply:
thanks for your comments! already changed everything